_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,327 @@
+we urge you to get in touch with the core developer team. You can do so over our
+[public forums]({% link index.md %}#participating) or privately by emailing *licensing@mdanalysis.org*.
+
+## Definitions
+
+### package license
+
+The license under which the MDAnalysis library is packaged and released. This may be different from
+but still compatible with the [source code license](#source-code-license).
+
+### source code license
+
+The license under which the MDAnalysis source code is released. This only accounts for the code itself
+and not any impact in licensing incurred from linking (i.e. `python importing`) to other libraries.
+
+### license compatibility
+
+Whether or not the terms of two licenses allows them to be compatible with each other.
+
+### copyleft
+
+Whether the terms of a given license enforces that other codes (even under other licenses) must be bound
+by the same conditions. See the [FSF definition](https://www.gnu.org/licenses/copyleft.en.html) for more information.
+
+## Acknowledgements
+
+We are very grateful for the administrative and legal support from our fiscal sponsor, [NumFOCUS][].
+
+This relicensing effort has been made possible in part by the CZI grant DAF2021-237663,
+grant DOI https://doi.org/10.37921/426590wiobus from the Chan Zuckerberg Initiative DAF,
+an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989).
+
+– The MDAnalysis [Core Developers][]
+
+[OSI]: https://opensource.org/osd
+[tldr legal]: https://tldrlegal.com/
+[SSI licensing]: https://www.software.ac.uk/resources/guides/choosing-open-source-licence
+[OSD]: https://opensource.org/osd
+[GPL v2]: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+[GPL v3]: https://www.gnu.org/licenses/gpl-3.0.en.html
+[LGPL v2.1]: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+[LGPL v3]: https://www.gnu.org/licenses/lgpl-3.0.en.html
+[MIT]: https://opensource.org/license/mit/
+[BSD 3.0]: https://opensource.org/license/bsd-3-clause/
+[Apache 2.0]: https://www.apache.org/licenses/LICENSE-2.0
+[contributors]: https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS
+[NumFOCUS]: https://www.numfocus.org
+[fasteners]: https://fasteners.readthedocs.io/en/latest/
+[mmtf-python]: https://github.com/rcsb/mmtf-python
+[py-filelock]: https://github.com/tox-dev/filelock
+[MMTF]: https://mmtf.rcsb.org/
+[44733fc]: https://github.com/MDAnalysis/mdanalysis/commit/44733fc214dcfdcc2b7cb3e3705258781bb491bd
+[MDAnalysis core library]: https://github.com/MDAnalysis/mdanalysis
+[previous relicensing post]: {{ site.baseurl }}{% post_url 2022-11-07-relicensing %}
+[rationale]: {{ site.baseurl }}{% post_url 2022-11-07-relicensing %}#rationale-for-license-change
+[consultation]:  {{ site.baseurl }}{% post_url 2022-11-07-relicensing %}#consultation-period-7th-november-until-5th-december-2022
+[release 2.6]: {{ site.baseurl }}{% post_url 2023-09-02-release-2.6.1.md %}
```

Comment:
```suggestion
[release 2.6]: {{ site.baseurl }}{% post_url 2023-09-02-release-2.6.1 %}
```

Fixes render:
![Uploading Screenshot 2023-09-25 at 12.45.06 pm.png…]()

Reply:
Yeah sorry that's my bad - silly copy paste typo. I still can't get jekyll to behave on my laptop so local testing isn't happening.